pkg/releaseutil/sorter_test.go

```diff
@@ -79,3 +79,29 @@ func TestSortByRevision(t *testing.T) {
 		return vi < vj
 	})
 }
+
+func TestReverse(t *testing.T) {
```

Comment:
@tiendc Thanks for adding the tests. The checks for each test will however be cached and only fail if the last test fails. Can you separate them into  different test functions?